src/scripts/moment-select/directives.js

```diff
@@ -16,8 +16,20 @@
         }
     ]);
 
+    angular.module('aif-add-suffix-filter', []).filter('aifAddSuffix', [
+        function () {
+            return function (input, suffix) {
```

Comment:
I did think of that, but there is no performance gain here, as the memoize cache comparing function will be exactly the same as the filter itself.